testapp/testapp/testmain/views.py

```diff
@@ -26,3 +29,23 @@ def payment_success(request):
 
 def payment_failure(request):
     return HttpResponse("Payment failed.")
+
+
+def create_test_payment(request):
```

Comment:
I think that this should do it for now... Maybe redirect() could be optional

Reply:
I think it's good as-is.